src/amount.h

```diff
@@ -36,8 +37,10 @@ inline bool MoneyRange(const CAmount& nValue) { return (nValue >= 0 && nValue <=
 class CFeeRate
 {
 private:
-    CAmount nSatoshisPerK; // unit is satoshis-per-1,000-bytes
+    CAmount nSatoshisPerK; //! unit is satoshis-per-PRECISION_MULTIPLIER-bytes
```

Comment:
See #7731. You cannot *2 without disruption. *10 requires more disruption, etc. Besides, this includes new tests that would fail qhen changed. Nobody will change the constant "without noticing".